WhatsApp/app/src/main/java/com/example/whatsapp/ui/fragments/home/chats/ChatsFragment.kt

```diff
@@ -86,6 +86,11 @@ class ChatsFragment : BaseFragment<FragmentChatsBinding>(), ActionMode.Callback
         //endregion
     }
 
+    override fun onPause() {
+        tracker?.clearSelection()
```

Comment:
Same situation here as in CallsFragment

Reply:
Update c34973a16b8752e1364bbf7904a2549e90ab874b